src/assets/style.css

```diff
@@ -0,0 +1,86 @@
+/* stylelint-disable */
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linter disable rule in your project. The linters are there to make sure your code follows the best practices, I'll suggest you remove this disable rule and fix the issue correctly. :+1: :+1: